example/cumulus-tf/iam.tf

```diff
@@ -1,11 +1,18 @@
 data "aws_cloudformation_export" "fake_s3_provider" {
   name = "cumulus-fake-s3-provider-bucket"
 }
+data "aws_cloudformation_export" "fake_s3_provider_alternate" {
```

Comment:
The updates to this file are all required to setup an alternate bucket for ingest testing.